app/screens/channel_info/index.js

```diff
@@ -39,6 +42,7 @@ function mapStateToProps(state) {
             isTeammateGuest = true;
             currentChannelGuestCount = 1;
         }
+        customStatus = getCustomStatus(state, teammateId) || {};
```

Comment:
same comment about the empty object and you should also check if the feature is enabled before calling the selector.